conf/ucd_sonic.config

```diff
@@ -0,0 +1,38 @@
+//UCD_SONIC config
+params {
+  config_profile_name = 'UCD_SONIC'
+  config_profile_description = 'University College Dublin Sonic HPC profile provided by nf-core/configs.'
+  config_profile_contact = 'Bruce Moran (@brucemoran)'
+  config_profile_url = 'https://www.ucd.ie/itservices/ourservices/researchit/researchcomputing/sonichpc/'
+  //max_memory = 1.MB //yes really, allocation made based on other factors and cannot be done via params.max_memory...
```

Comment:
This is not an allocation but rather a cap which needs to be defined at a pipeline level - setting to 1.MB will fail! I guess you may have worked that our so I would remoce teh commend

Reply:
OK, my mistake

Reply:
```suggestion
```